content/node-runners/setup/docker.md

```diff
@@ -51,60 +47,12 @@ Docker Desktop for Windows uses Windows-native Hyper-V virtualization and networ
-2.  Open a command-line terminal and type the following command:
-
-```bash
-docker run --cap-add NET_ADMIN -p 4449:4449 -p 41920-42075:41920-42075/udp -p 61920-62075:61920-62075/udp --name myst -v $YOUR_MYSTERIUM_DIR:/var/lib/mysterium-node mysteriumnetwork/myst:latest --experiment-natpunching=false --p2p.listen.ports=41920:42075 service --agreed-terms-and-conditions --wireguard.listen.ports=61920:62075 wireguard
-```
-
-**_Note 1:_** Replace `$YOUR_MYSTERIUM_DIR` with the path where you'd like to store the node's configuration and keystore files, e.g.
-
-```bash
-export YOUR_MYSTERIUM_DIR=~/.mysterium
-```
-
-**_Note2:_** By adding `--agreed-terms-and-conditions` command line option you accept our Terms & Conditions.
-
-**_Note3:_** Use  Docker detached mode by adding the option `--detach` or `-d`.
-
-**_Note4:_** Disable NAT hole punching by adding `--experiment-natpunching=false` option to use port forwarding.
-
-**_Recommended port mappings:_**
-
-- -p 4449:4449 The port to run Node UI on (default: 4449)
-- -p 61920-62075:61920-62075/udp Range of WireGuard listen ports (e.g. 61920:62075)
-- -p 41920-42075:41920-42075/udp Range of P2P listen ports (e.g. 41920:42075)
-- --experiment-natpunching=false Disables NAT hole punching
-
-You can use **_different values_** to change listed services' **_port_** numbers for clarity and convenience.
-
-Your Windows Docker node is now ready to serve Mysterium Network users!
-
-Make sure to use volumes as in the example above to persist your node's identity through container and host system restarts or node image upgrades.
-
-## Docker on Windows and MacOS (Updated and simplified way)
-
 ### Known limitations:
 
 - Because of the way networking is implemented in Docker Desktop for Windows/Mac, you cannot see a docker0 interface on the host. This interface is actually within the virtual machine.
-- Docker Desktop for Windows/Mac can’t route traffic to Linux containers. 
+- Docker Desktop for Windows/Mac can’t route traffic to Linux containers.
 - The docker (Linux) bridge network is not reachable from the Windows/Mac hosts.
 
-In previous Docker implementations, it was mandatory to set up Port forwarding so that ports exposed from Linux are forwarded to the host.
-Another requirement was to map the necessary custom port(s) from the router to your local machine: [YOURPUBLICIP]:PORT to -> [YOURLOCALNETWORK_IP]:PORT so it becomes publicly exposed via the router. This was necessary to override certain limitations and make it usable.
-
-### Solution
-
-The Mysterium team created a workaround to bypass these recognized problems and known limitations, implementing a STUN (Simple Traversal of UDP over NAT or Session Traversal Utilities for NAT) server to automatically determine the IP address and correct port(s) allocated to it by a NAT. This helped minimize the effort required, simplifying the process as much as possible for Windows & Mac users.
```

Comment:
This is not a workaround - this is a standard approach for it. Calling it a workaround makes people feel like we are doing some bad product.